wiki/Submission/Beatmap_Ranking_Procedure/en.md

```diff
@@ -1,44 +1,25 @@
+Note: In the case of a minor change to a nominated beatmapset, then the BN can nominate the beatmapset again immediatly, but in the case of a veto, a decision from the QAT needs to be done before the map is allowed to be nominated again.
 
-Qualified / Ranked ![Heart icon](/wiki/shared/icon/heart.gif "Heart icon")
+Qualified and Ranked Sections
 -------------------------------------------------------------------------
 
-Once a map is bubbled, a BN can qualify it if the map meets the two criteria listed above. The reason two approval team members are required to check a map is because human beings make mistakes, and this greatly reduces the times of that a map makes it through the ranking system with a drastic error. If a map gets qualified but approval team members or anyone else found it missed a drastic error (e.g. A 5mb video is included in the `.osz` that is not used), then it might get disqualified. After a map got into the qualified section for a certain amount of time (usually about one week) with no errors been found, it will automatically get ranked afterwards.
-
-Disqualified ![Heart-broken icon](/wiki/shared/icon/broken-heart.gif "Heart-broken icon")
-----------------------------------------------------------------------------------
-
-People make mistakes. Therefore, if a map makes it through the approval system with a drastic error, then it might become temporarily disqualified. If your map gets disqualified, you need to address the unrankable issues in order to fix what was wrong, so that your map may become re-qualified and be enjoyed as quickly as possible.
-
-Note: You'll sometimes be asked to change things you do not want to change. Sometimes you may be able to get away without changing it, but if half the Beatmap Nomination Group agree that you should change it, then let go of your pride and just fix it.
-
-Approved ![Fire icon](/wiki/shared/icon/flame.gif "Fire icon")
-------------------------------------------------------------
-
-Following up from [ranked](#qualified---ranked), a BN can approve these beatmaps if they are longer than 5 minutes long (aka [marathon](/wiki/Glossary/#marathon) beatmaps) and/or that do not follow the [ranking criteria](/wiki/Ranking_Criteria) but are considered "okay" by a BN.
+Once a beatmapset reaches two nominations, it will reach the Qualified section. While in the Qualified section, the map will gain a leaderboard, but will not award [Performance Points](/wiki/Performance_Points). The reason two Beatmap Nominators are required to check a beatmapset is because human beings make mistakes, and this greatly reduces the likelyhood of a map making it through the ranking system with a drastic error. If a beatmap gets qualified but an unrankable issue is found or improvements to the beatmap have been proposed and accepted, then the beatmapset will be disqualified by a QAT member. This resets the nominations on the map. 
```

Comment:
`likelyhood` -> `likelihood`
```suggestion
Once a beatmapset reaches two nominations, it will reach the Qualified section. While in the Qualified section, the map will gain a leaderboard, but will not award [Performance Points](/wiki/Performance_Points). The reason two Beatmap Nominators are required to check a beatmapset is because human beings make mistakes, and this greatly reduces the likelihood of a map making it through the ranking system with a drastic error. If a beatmap gets qualified but an unrankable issue is found or improvements to the beatmap have been proposed and accepted, then the beatmapset will be disqualified by a QAT member. This resets the nominations on the map. 
```